baseplate/lib/thrift_pool.py

```diff
@@ -170,7 +171,7 @@ def __init__(
 
         self.size = size
 
-        self.pool = queue_cls
+        self.pool = queue_cls()
```

Comment:
I can't find anywhere internally that `queue_cls` is being passed in as an argument, but just wanted to call out that this is technically a backwards-incompatible change.

Reply:
You are correct. Only 3 services are on 2.4.x+ and none of them take advantage of this (this is primarily used for tests)